lib/game/round.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Reactor.Game.Round do
+  use GenServer
+
+  @colors ["red", "green", "yellow", "blue", "orange", "pink"]
+
+  def start_link(game_id, users) do
+    GenServer.start_link(__MODULE__, [game_id, users])
+  end
+
+  def submit(_hello, _world) do
+    {:ok}
+
+    # update_state
+    # check state
+    # if everyone submitted
+    # emit event
+  end
+
+  ##Server Callbacks
+
+  def init([game_id, users]) do
+    #get random colors
+    #set it in state
+    #emit to game_event_manager
+    colors = Enum.take_random(@colors, 4)
```

Comment:
sorry by Game I meant the Game Process (the thing spawning a Round). I don't the setting of colors to be async at least I don't think I do and if that's the case, Game is going to be waiting for the {:ok, pid} response anyway. I understand that moving it out into another function might be better but I don't understand how it helps the problem of "new process is sync so nothing else can be started via the supervisor until this has completed"